src/app/components/restaurants/RestaurantsMenu.jsx

```diff
@@ -42,14 +42,14 @@ class RestaurantsMenu extends React.Component {
         <div className="menu-content">
           <div className="search-bar">
             <SearchInput
-              placeholder={this.state.placeholder}
+              placeholder={this.state.isPlaceholderVisible ? 'Wyszukaj...' : ''}
               onChange={(event) => this.changeSearchInput(event)}
               onClick={this.hiddenPlaceholder}
               onBlur={this.onSearchBlur}
             />
           </div>
           <div className="button">
-            <FlatButton onClick={(event) => this.changeMenuLayout(event)}>
+            <FlatButton onClick={() => this.changeMenuLayout()}>
```

Comment:
Brat bo robiłem to na wzór tego z rejestratorki, a dopiero dzisiaj poprawiałem to na rejestratorce, wtedy jeszcze nie widzialem, sorki